src/mcts/search.cc

```diff
@@ -956,7 +956,7 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
         ++possible_moves;
       }
       const float Q = child.GetQ(0);
-      const float tanhU = tanh(child.GetU(puct_mult)) - fpu;
+      const float tanhU = tanh(child.GetU(puct_mult)) - (child.GetQ(fpu) - Q);
```

Comment:
IMO the fpu term should be treated as if it was a U value so it should be inside the tanh bracket?

Reply:
I think you're right. I'll move that parenthesis.